app/steps/ui/executors/contactdetails/index.js

```diff
@@ -32,6 +33,7 @@ class ExecutorContactDetails extends ValidationStep {
     }
 
     * handlePost(ctx, errors) {
```

Comment:
Can you add a unit test for this method.

It doesn't appear that this class has any unit tests so you'll need to create the unit test file...don't worry about unit testing the methods that you've not changed.